### PR TITLE
[sc-131251] Add option to change maxCount value

### PR DESCRIPTION
**Pull request by alexbourret**

### CODE REVIEW

**Review by liamlynch-data — CHANGES_REQUESTED** [review]
Have tested it on all the relevant components and looked at the code. 

For the Recorded data type it's all good - the `max_count` param does what is expected and changes the numbers of results coming back. 

For the Interpolated and Plot data type `maxCount` query param does not do anything at the API level of OsiSoft (it is not active for these endpoints) so it has no effect in the plugin either, so I don't think we should be adding it here.

For the Interpolated type, I think the interpolation params directly determine the number of results per path within the time range given, so that is all we need I think.

For Plot, the `intervals` param (plural) determines how many slices of the time dimension sampling occurs on so directly determines the number of results (if I understand correctly - see [here](https://docs.aveva.com/bundle/pi-web-api-reference/page/help/controllers/stream/actions/getplot.html))

So I think we should remove the `max_count` for both Interpolated and Plot and maybe add in the `intervals` one for Plot  at some point (if we think people will need it).

File: python-connectors/pi-system_piwebapi-toolbox/connector.py [hunk 1]
```diff
@@ -61,7 +65,7 @@ def generate_rows(self, dataset_schema=None, dataset_partitioning=None,
                 end_date=self.end_time,
                 interval=self.interval,
                 sync_time=self.sync_time,
-                endpoint_type="AF"
+                endpoint_type="AF", max_count=self.max_count
```
[the remark this answers is not in the document]
Comment (alexbourret) [reply python-connectors/pi-system_piwebapi-toolbox/connector.py:68]: 👍 fixed in [3e75509](https://github.com/dataiku/dss-plugin-osisoft/pull/35/commits/3e755093ad692bc653b7f9e9b7aba9faf304f039)

**Comment by alexbourret** [comment]
maxCount selector visibility is now fixed with this commit : [c44678d](https://github.com/dataiku/dss-plugin-osisoft/pull/35/commits/c44678d823fcc533dff222f0f912c80d97dc8cd6)

**Comment by liamlynch-data** [comment]
I took a look at the changes in the destination branch (feature/sc-131263-add-time-span-filters-to-ef-search), all looks good to me.

**Review by liamlynch-data — APPROVED** [review]

**Pull request merged**